skimage/filters/thresholding.py

```diff
@@ -727,14 +727,10 @@ def threshold_triangle(image, nbins=256):
     width /= norm
 
     # Maximize the length.
-    d = peak_height * arg_low_level - width * hist[arg_low_level]
```

Comment:
I come up to the same result. I'm +1 to add the sentence you suggest as a comment